docs/pre-release/install/upgrade-tabs.js

```diff
@@ -97,6 +99,35 @@ export default function SimpleTabs() {
         }
         </CodeBlock>
       </TabPanel>
+      <TabPanel value={value} index={2}>
+        <CodeBlock>
+        {
+          '# Make sure you run the following from Powershell as Administrator' +
+          '# 1. Download the latest windows zip containing telepresence.exe and its dependencies (~50 MB):' +
+          '\n' +
+          'curl -fL https://app.getambassador.io/download/tel2/windows/amd64/latest/telepresence.zip -o telepresence.zip' +
+          '\n \n' +
+          '# 2. Unzip the zip file to a suitable directory + cleanup zip' +
+          '\n' +
+          'Expand-Archive -Path telepresence.zip' +
+          '\n' +
+          'Remove-Item \'telepresence.zip\'' +
+          '\n' +
+          'cd telepresence' +
+          '\n \n' +
+          '# 3. Run the install-telepresence.ps1 to install telepresence\'s dependencies. It will install telepresence to' +
+          '\n' +
+          '# C:\\telepresence by default, but you can specify a custom path $path with -Path $path' +
```

Comment:
hmm this sentence doesn't track very well imho -- how about something like `but you can specify a custom path by passing in -Path C:\\my\\custom\\path`